test/errors/invalidNew-brief.good

```diff
@@ -0,0 +1,2 @@
+error in invalidNew.chpl:3: cannot use memory management strategy 'owned' with non-class type 'E'
+error in invalidNew.chpl:3: invalid use of 'new' with type 'E', which is neither a class nor a record
```

Comment:
Nothing needs to change here, but for future reference, note that it's been proposed that we add support for `new t()` for non-class/record types for consistency / to support generic programming.  E.g., `var x = new int(42);` as an alternative to `var x: int = 42;`.